### PR TITLE
Feature/claudia hello world

### LINKED ISSUE #7: Lambda "Hello World"
Get a "Hello World" endpoint working on Lambda using the deployment framework chosen in #6.

If we haven't already, this will require that we choose or create an AWS account and set up credentials on it.

### PR DESCRIPTION
**Pull request by mattdelsordo**
## Overview

Adds a basic "hello world" for AWS Lambda using Node.

### Notes

Requires proper AWS credentials set up to test.

## Testing Instructions

 * Run `npm install` or `yarn install` to grab dependencies.
 * Running `npm run start` or `yarn start` will create a new Lambda on AWS Lambda called 'hello-world-${USER}'. Output should look like the following:
~~~~
$ claudia create --region us-east-1 --role lambnik-tiler-dev --api-module main --name hello-world-${USER}
loading package config
packaging files
packaging files npm pack /home/mdelsordo/azavea/tilegarden/node_lambnik
npm notice 
npm notice 📦
npm notice === Tarball Contents === 
npm notice 571B package.json
npm notice 350B main.js     
npm notice === Tarball Details === 
npm notice name:          tilegarden                              
npm notice version:       1.0.0                                   
npm notice filename:      tilegarden-1.0.0.tgz                    
npm notice package size:  611 B                                   
npm notice unpacked size: 921 B                                   
npm notice shasum:        xxxxxxxxxxxxxxxxxxxxxxxxx
npm notice integrity:     xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
npm notice total files:   2                                       
npm notice 
packaging files npm install -q --no-audit --production
npm WARN tilegarden@1.0.0 No repository field.

added 1 package from 1 contributor in 2.698s
validating package
validating package      npm dedupe -q --no-package-lock
zipping package
initialising IAM role
initialising IAM role   iam.getRole     RoleName=lambnik-tiler-dev
creating Lambda
creating Lambda lambda.createFunction   FunctionName=hello-world-mdelsordo
creating Lambda lambda.setupRequestListeners
creating version alias
creating version alias  lambda.updateAlias      FunctionName=hello-world-mdelsordo      Name=latest
creating version alias  lambda.setupRequestListeners
creating version alias  lambda.createAlias      FunctionName=hello-world-mdelsordo      Name=latest
creating version alias  lambda.setupRequestListeners
creating REST API
creating REST API       apigateway.createRestApi        name=hello-world-mdelsordo
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       sts.getCallerIdentity
creating REST API       apigateway.getRestApi   restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.updateRestApi        restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.getResources restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.getGatewayResponses  restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putMethod    authorizerId=undefined  resourceId=jhp1tlrgse   restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putIntegration       resourceId=jhp1tlrgse   restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putMethodResponse    resourceId=jhp1tlrgse   restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putIntegrationResponse       resourceId=jhp1tlrgse   restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putMethod    resourceId=jhp1tlrgse   restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putIntegration       resourceId=jhp1tlrgse   restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putMethodResponse    resourceId=jhp1tlrgse   restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putIntegrationResponse       resourceId=jhp1tlrgse   restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.createResource       parentId=jhp1tlrgse     pathPart={name} restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putMethod    authorizerId=undefined  resourceId=pth2w9       restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putIntegration       resourceId=pth2w9       restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putMethodResponse    resourceId=pth2w9       restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putIntegrationResponse       resourceId=pth2w9       restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putMethod    resourceId=pth2w9       restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putIntegration       resourceId=pth2w9       restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putMethodResponse    resourceId=pth2w9       restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putIntegrationResponse       resourceId=pth2w9       restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.createResource       parentId=jhp1tlrgse     pathPart=echo   restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putMethod    authorizerId=undefined  resourceId=4a5ia1       restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putIntegration       resourceId=4a5ia1       restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putMethodResponse    resourceId=4a5ia1       restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putIntegrationResponse       resourceId=4a5ia1       restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putMethod    resourceId=4a5ia1       restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putIntegration       resourceId=4a5ia1       restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putMethodResponse    resourceId=4a5ia1       restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.putIntegrationResponse       resourceId=4a5ia1       restApiId=ot36sxu4a5
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
creating REST API       apigateway.createDeployment     restApiId=ot36sxu4a5    stageName=latest
creating REST API       apigateway.setupRequestListeners
creating REST API       apigateway.setAcceptHeader
saving configuration
{
  "lambda": {
    "role": "lambnik-tiler-dev",
    "name": "hello-world-mdelsordo",
    "region": "us-east-1",
    "sharedRole": true
  },
  "api": {
    "id": "ot36sxu4a5",
    "module": "main",
    "url": "https://ot36sxu4a5.execute-api.us-east-1.amazonaws.com/latest"
  }
}
Done in 10.61s.
~~~~
* Optional: edit one of the returned strings in main.js and run `yarn update` to update the lambda. It should give you very similar output to `yarn start`.
* Visit the provided URL, you should see a `Hello, world!`. You can also append `/:name` to get `Hello, ${name}`, or `/echo` followed by any request to get returned the request. 

Resolves #7

### CODE REVIEW

**Review by KlaasH — COMMENTED** [review]
Looks good so far.  But this just gets us a Lambda function that we can invoke directly (via Claudia or in the console), right?  No HTTP-accessible endpoint?  I think it's worth adding a "hello world"-level endpoint as part of this as well.  Am I right that Claudia API Builder should be able to get us that pretty easily?

Also, I think the `claudia.json` file should be local-only (i.e. not checked in and instead added to `.gitignore`).  It gets created by the `start` command, and when I try to run `yarn start` on this branch as-is it returns an error because the file already exists.  I think also for this stage, before we've added more parameterized config, it would make sense to make the `--name` argument in the `start` command be `hello-world-${USER}`.  That's a development-mode pattern, namespacing by username, but I think it suits the present purpose.

**Comment by mattdelsordo** [comment]
Sounds like a good idea, I'll add in the API builder code. I was actually about to ask about .gitignoring claudia.json, I've ran into trouble with it too when switching branches and whatnot.

**Review by KlaasH — APPROVED** [review]
Looks great! :+1:

File: node_lambnik/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "files": [
+    "main.js"
+  ],
+  "scripts": {
+    "start": "claudia create --region us-east-1 --role lambnik-tiler-dev --api-module main --name hello-world-${USER}",
+    "test": "claudia test-lambda",
+    "deploy": "claudia update"
+  },
+  "devDependencies": {
+    "claudia": "^5.0.0"
```
Comment (KlaasH) [inline node_lambnik/package.json:17]: ~I believe this should be in `dependencies` rather than `devDependencies`.~  Whoops, nope (as discussed just now).

**Pull request merged**